src/components/my-history/custom-my-history.vue

```diff
@@ -17,11 +17,15 @@
             data-test="my-history-query"
             :suggestion="suggestion"
             suggestionClass="x-suggestion"
+            class="hover:x-no-underline"
           >
             <HistoryIcon class="max-desktop:x-icon-lg" />
 
-            <div class="x-flex x-flex-col x-gap-2">
-              <p>{{ suggestion.query }}</p>
+            <div class="x-group x-flex x-flex-col x-gap-2">
+              <p class="group-hover:x-underline">{{ suggestion.query }}</p>
+
+              <HistoryQueryFilters class="x-w-192" :suggestion="suggestion"></HistoryQueryFilters>
```

Comment:
```suggestion
              <HistoryQueryFilters class="x-w-192" :suggestion="suggestion"/>
```

Also, the list of filters could be have more width here, bump it to something around 250-320